venues/ICLR.cc/2020/Conference/python/buddy_ac.py

```diff
@@ -0,0 +1,133 @@
+
+    conference_id = conference.get_id()
+
+    buddy_ac_parent_group = client.get_groups(regex = conference_id + '/' + buddy_ac_parent_group_name)
+
+    if not buddy_ac_parent_group:
+        ac_group = client.get_group(conference.get_area_chairs_id())
+
+        buddy_ac_group = client.post_group(
+            openreview.Group(
+                id = conference_id + '/' + buddy_ac_parent_group_name,
+                readers = [
+                    conference_id,
+                    conference.get_program_chairs_id(),
+                    conference.get_area_chairs_id()
+                ],
+                signatories = [conference_id + '/' + buddy_ac_parent_group_name],
+                signatures = [conference_id],
+                writers = [conference_id],
+                web = 'buddyAreachairWebfield.js',
+                members = ac_group.members
+            )
+        )
+
+    map_meta_review_invitations = {invitation.id.split('Paper')[1].split('/')[0]: invitation for invitation in openreview.tools.iterget_invitations(client, regex = conference_id + '/Paper[0-9]+/-/Meta_Review')}
+
+    map_paper_ac_groups = {group.id.split('Paper')[1].split('/')[0]: group for group in openreview.tools.iterget_groups(client, regex = conference_id + '/Paper[0-9]+/Area_Chairs$')}
+
+    submissions = conference.get_submissions()
+    for paper in submissions:
+        paper_number = str(paper.number)
+        individual_buddy_group_id = conference_id + '/Paper' + paper_number + '/' + buddy_ac_individual_group_name
+
+        ## Create buddy ac group for this paper
+        individual_buddy_group = client.post_group(
+            openreview.Group(
+                id = individual_buddy_group_id,
+                readers = [
+                    conference_id,
+                    conference.get_program_chairs_id(),
+                    individual_buddy_group_id],
+                nonreaders = [conference_id + '/Paper' + paper_number + '/Authors'],
+                signatories = [individual_buddy_group_id],
+                signatures = [conference.get_id()],
+                writers = [conference.get_id()],
+                members = []
+            )
+        )
+
+        # Add paper's buddy AC as member to paper's AC group
+        paper_ac_group = map_paper_ac_groups[paper_number]
+        client.add_members_to_group(paper_ac_group, individual_buddy_group_id)
+
+        # Add paper's buddy AC as non-invitee for paper's meta-review invitation
+        paper_meta_rev_invitation = map_meta_review_invitations[paper_number]
+        paper_meta_rev_invitation.noninvitees
+
+        ## Post AC & AC-Buddy only Comment invitation for this paper
+        ac_conversation_invitation = openreview.Invitation(
+            id = conference_id + '/Paper' + paper_number + '/-/Area_Chair_Only_Comment',
+            readers = [
+                conference.get_area_chairs_id(number = paper_number),
```

Comment:
using this group should be enough, right? you don't need to add the buddy group id

Reply:
Correct. Fixed.